### PR TITLE
Upgrade to Vaadin 23 + Hilla 1

### PR DESCRIPTION
**Pull request by Artur-**

### CODE REVIEW

**Comment by tarekoraby** [comment]
I still get an error in the Hilla Master-Detail view:
```
VM846:4 There seems to be an error in Vaadin Grid:
Cannot read properties of undefined (reading 'hasEnsureSubCacheQueue')
Please submit an issue to https://github.com/vaadin/flow-components/issues/new/choose
```

File: pom.xml [hunk 1]
```diff
@@ -73,15 +74,25 @@
                 <type>pom</type>
                 <scope>import</scope>
             </dependency>
+            <dependency>
```
Comment (manolo) [inline pom.xml:77]: I think you can remove vaadin-bom since hilla already has it

Comment (Artur-) [reply pom.xml:77]: Sounds like an internal (problem) of Hilla that will be resolved at some point?

**Comment by Artur-** [comment]
> I still get an error in the Hilla Master-Detail view:
> 
> ```
> VM846:4 There seems to be an error in Vaadin Grid:
> Cannot read properties of undefined (reading 'hasEnsureSubCacheQueue')
> Please submit an issue to https://github.com/vaadin/flow-components/issues/new/choose
> ```

Which is https://github.com/vaadin/flow-components/issues/2755 so nothing to fix here

**Review by tarekoraby — APPROVED** [review]

**Comment by tarekoraby** [comment]
I'll merge this and we can address any further fixes or enhancements in separate PRs.

**Pull request merged**